### ISSUE TITLE
I2c Recovery Example application not available for STM32

### ISSUE THREAD

**Issue by ShubhadaHarkut-eaton** [zephyrproject-rtos/zephyr#76287]
I2c Recovery functions to prevent lockup is not used in any sample or driver code for stm32 controllers (stm32h743zi , stm32u575zi )

**Comment by ttmut** [comment]
I do not think that this should be considered a bug. Still, you could use I2C shell to test the recovery function.

**Comment by jhedberg** [comment]
@erwango any thoughts?
